node/_tools/test.ts

```diff
@@ -47,6 +47,7 @@ for await (const file of dir) {
           "-A",
           "--quiet",
           "--unstable",
+          "--no-check",
```

Comment:
This change should be done in a separate PR - I think there's no need to type check on every run if we did it once using `deno cache` command

Reply:
I'm in favor of this change. This command checks the typing of `node/_tools/require.ts` hundreds of times, and that's redundant

Reply:
Correct, each subprocess takes at least 2s.